src/main/java/org/kohsuke/github/GitHubClient.java

```diff
@@ -632,11 +651,15 @@ private static IOException interpretApiError(IOException e,
 
     private static void logRetryConnectionError(IOException e, URL url, int retries) throws IOException {
         // There are a range of connection errors where we want to wait a moment and just automatically retry
+        long sleepTime = minRetryInterval;
```

Comment:
I suggest setting `minRetryInterval` and `maxRetryInterval` via `Integer.getInteger` property here.